src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -47,6 +49,40 @@ public Optional<String> getDerivedTableName(UUID entityTypeId) {
       .fetchOptional(derivedTableNameField);
   }
 
+  public List<EntityTypeColumn> fetchNamesForSingleCheckbox(UUID entityTypeId) {
```

Comment:
Rather than fetch only the checkbox custom fields (implying that we'll need to do a separate method for radios vs text inputs vs etc, where each is `select ..., ... from ... where type = ______`), I'd do 1 method for all custom fields (so that we only do 1 query of `select ..., ..., type from ...`), then iterate over the results of that query, dispatching to different methods, depending on the type of the custom field, skipping+logging types that we don't support yet